benchmark/sirun/startup/meta.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "startup",
+  "run": "node startup-test.js",
```

Comment:
Does it matter if we use the same script or not?

```json
{
  "name": "startup",
  "cachegrind": true,
  "variants": {
    "control": { "run": "node -e ''" },
    "with-tracer": { "run": "node -e 'require('../../../').init()" }
  }
}
```


It actually doesn't matter at all, one way or the other, just curious if there is a reason to choose one approach over the other (mostly for my own knowledge when I write some scenarios)

Reply:
It doesn't matter. This was purely personal preference. I did want to use a file though, rather than `-e` to be at least a *little* closer to what users actually do. The `-e` environment isn't _quite_ the same as running a script.

Reply:
makes sense